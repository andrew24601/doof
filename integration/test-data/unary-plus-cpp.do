// Demonstrates unary plus support (+x) in C++ backend

function main(): int {
    let x = +1;
    let y = +3.5;
    println(string(x) + " " + string(y));
    return 0;
}
