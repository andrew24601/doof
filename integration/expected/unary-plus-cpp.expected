1 3.500000
